mw-webapp/src/dataAccessLogic/DayReportDAL.ts

```diff
@@ -9,6 +9,28 @@ import {DayReport} from "src/model/businessModel/DayReport";
 import {DayReportDTOWithoutUuid, DayReportService} from "src/service/DayReportService";
 import {WayService} from "src/service/WayService";
 import {DateUtils} from "src/utils/DateUtils";
+import {unicodeSymbols} from "src/utils/unicodeSymbols";
+
+/**
+ * DayReport props
+ */
+interface DayReportProps {
```

Comment:
why are you keeping props interface in DAL layer?

Reply:
Deleted